### DISCUSSION TITLE
Running on Amazon Linux 2

### DISCUSSION THREAD

**Discussion by dayunbao** [bmaltais/kohya_ss#1544] [Q&A]
Has anyone had any success getting this to run on Amazon Linux 2?  I'm using a g4dn.xlarge instance type, by the way.  I keep getting this error when trying to run BLIP captioning:
`
RuntimeError: The size of tensor a (80) must match the size of tensor b (800) at non-singleton dimension 0
`
I'm at a bit of a loss here.  Any help anyone can offer would be much appreciated!

**Comment by wutipong** [comment]
I think I have this issue on my Windows machine. Basically if number of beams is larger than 1, the error would show up.

Can you try setting it to 1 and run the captioning again ?
